client/my-sites/plugins/index.js

```diff
@@ -82,8 +82,12 @@ module.exports = function() {
 			pluginsController.eligibility
 		);
 
-		page.exit( '/plugins*',
-			pluginsController.resetHistory
-		);
+		page.exit( '/plugins*', ( context, next ) => {
+			if ( 0 !== page.current.indexOf( '/plugins' ) ) {
```

Comment:
Makes sense, added in f6667c4 (also added it to the `exit()` path as well just in case)